source/logging.rst

```diff
@@ -0,0 +1,67 @@
+.. _doc_routinator_logging:
+
+Logging
```

Comment:
You might want to include the options to configure logging via the config file here. Particularly in server mode this is the most likely way people will want to configure this since fiddling with command line options is kind of awkward for init scripts and whatnot.